### ISSUE TITLE
Instructions unclear: How do you host UBer.js on a github website page?

### ISSUE THREAD

**Issue by Zelkicks** [luphoria/UBer#8]

**Comment by Zelkicks** [comment]
I put it as a bookmark and a message does pop up, but when I put in a website nothing happens.

**Comment by Zelkicks** [comment]
Also, all websites refuse to connect :/

**Comment by luphoria** [comment]
Proxy is currently down

**Comment by Schoolstar14** [comment]
is it still down? Cuz I can't get on

**Comment by ok767** [comment]
its fine for me
only like some websites go weirdo mode

**Comment by luphoria** [comment]
> is it still down? Cuz I can't get on

Yes, I apologize for the loooong downtime. But, my server is having many issues trying to keep up with the many things I am doing on it. I will do a test run with Ultraviolet, and if my server handles it OK, then I will implement it into UBer :+1:
